components/Firebird/Deployments/ExistingDeployment/SelectedDeployment/StatusStream.tsx

```diff
@@ -20,14 +20,24 @@ export default function DeploymentStatusStream({
 }) {
   if (status === DeploymentStatus.ACTIVE) return
 
-  const eventDisplayNames = {
-    [DeploymentEvents.INIT_KEY_PAIR]: "Initializing Key Pair",
-    [DeploymentEvents.CREATE_INSTANCE]: "Creating Instance",
-    [DeploymentEvents.WAIT_INSTANCE_READY]: "Starting Instance",
-    [DeploymentEvents.INSTALL_KWILD]: "Installing Kwil Daemon",
-    [DeploymentEvents.REGISTER_DOMAIN]: "Registering Domain",
-    [DeploymentEvents.FINALIZE_DEPLOYMENT]: "Finalizing Deployment",
-  }
+  // Display appropriate events based on event stream events
+  const eventDisplayNames =
+    status === DeploymentStatus.STARTING
```

Comment:
The nested ternary is quite hard to read.

Could you just create a utility function somewhere with a switch and then set it `eventDisplayNames` inside the functional component?

E.g.

```typescript
// utility function outside your functional component
const getEventDisplayNames = (status: DeploymentStatus) => {
  switch (status) {
    case DeploymentStatus.STARTING:
      return {
        [DeploymentEvents.START_INSTANCE]: "Starting Instance",
      };
    case DeploymentStatus.STOPPING:
      return {
        [DeploymentEvents.STOP_INSTANCE]: "Stopping Instance",
      };
    default:
      return {
        [DeploymentEvents.INIT_KEY_PAIR]: "Initializing Key Pair",
        [DeploymentEvents.CREATE_INSTANCE]: "Creating Instance",
        [DeploymentEvents.WAIT_INSTANCE_READY]: "Starting Instance",
        [DeploymentEvents.INSTALL_KWILD]: "Installing Kwil Daemon",
        [DeploymentEvents.REGISTER_DOMAIN]: "Registering Domain",
        [DeploymentEvents.FINALIZE_DEPLOYMENT]: "Finalizing Deployment",
      };
  }
};

// and then inside `StatusStream`: 
const eventDisplayNames = getEventDisplayNames(status);
```